sdk/typescript/src/signers/txn-data-serializers/local-txn-data-serializer.ts

```diff
@@ -533,3 +561,7 @@ export class LocalTxnDataSerializer implements TxnDataSerializer {
     throw new Error(`Unsupported transaction type ${tx}`);
   }
 }
+
+export function shouldUseIntentSigning(version?: RpcApiVersion): boolean {
+  return version?.major === 0 && version?.minor >= 18;
```

Comment:
@666lcz is there a way to mock the version here in CI testing? I ran e2e test locally and see the version is returning 0.17. the testing should mock this value and return true.